src/test/codewhisperer/util/crossFileContextUtil.test.ts

```diff
@@ -123,6 +123,8 @@ describe('crossFileContextUtil', function () {
     })
 
     describe('partial support - control group', function () {
+        const fileExtLists: string[] = []
```

Comment:
yep, we don't have partial support language for now since we are now dialing up `.py` `.js` `.jsx` `.ts` `.tsx`

Reply:
we only have the above languages + `.java` for crossfile feat as of now

Reply:
Then I think we should just remove these two tests, the full support test is suffice, unless you want to keep it as a placeholder for future languages then that's ok.

Reply:
I would add a comment if for a placeholder.

Reply:
sure I will add a comment there since i prefer keeping it since we will still need it when we start support of other langs. Will remove it if we don't need it anymore